akanda/nova/neutronv2/api.py

```diff
@@ -61,55 +64,72 @@ def allocate_for_instance(self, context, instance, **kwargs):
             # pre-allocated port we also remove it from this set.
             available_macs = set(hypervisor_macs)
         neutron = neutronv2.get_client(context)
-        LOG.debug(_('allocate_for_instance() for %s'),
-                  instance['display_name'])
-        if not instance['project_id']:
+        LOG.debug('allocate_for_instance()', instance=instance)
```

Comment:
OpenStack stopped adding translation wrappers to debug messages during the Juno cycle.   I think it is a pattern we should continue to follow.